pylmod/base.py

```diff
@@ -13,15 +13,15 @@
 
 class Base(object):
     """
-    The Base class provides the transport for accessing MIT LM web service.
+    The Base class provides the transport for accessing MIT LMod web service.
 
     The Base class implements the functions that underlie the HTTP calls to
-    the LM web service.  It shouldn't be instantiated directly as it is
+    the LMod web service.  It shouldn't be instantiated directly as it is
     inherited by the classes that implement the API.
 
     Attributes:
-        cert: The certificate used to authenticate access to LM web service
-        urlbase: The URL of the LM web service
+        cert: The certificate used to authenticate access to LMod web service
```

Comment:
You should add types to attributes, i.e. `cert (unicode): ...`. Also, we should mention we are expecting the string path to the certificate, not the actual file handler or the string of the actual certificate.